engine-q/virtual_environments/README.md

```diff
@@ -0,0 +1,45 @@
+# Virtual environment scripts
+
+The scripts in this directory activate virtual environments for Conda environments.  They follow the pattern described in the [Nushell 0.32
+changelog](https://www.nushell.sh/blog/2021-06-01-nushell_0_32.html#environment-loading-lily-mara):
+
+```
+$ load-env (activate some-env)
+```
+
+A custom command (`activate`) creates a table with environment variables and
+`load-env` is used to load it into the shell's scope.
+
+In most cases, deactivation is a matter of restoring the PATH variable to the
+state before activating the env and removing any additional variables. There are
+no environment-specific elements to this, which is why the same deactivation
+script can be used for deactivating any environment of a specific type.  For
+example, `source conda_deactivate.nu` will deactivate any Conda env, there are
+no input parameters.
+
+## Expected Usage
+
+1. Source an activation script in your `config.nu`. For example, `conda.nu`.
+   You'll then have the `conda-env` command available.
+2. Create an alias for sourcing the appropriate deactivation script:
+   `alias conda-deactivate = source /path/to/conda_deactivate.nu`
+3. Activate with `load-env (conda-env env-name)`. You might want to define some
+   shorter aliases for both commands, if typing that every time seems like a
+   hassle.
+4. If you're using [Starship](https://starship.rs/), your prompt should reflect the activated env.
+5. When done, deactivate with your alias: `conda-deactivate`.
+
+Look at the script files to find the exact command name for creating an environment table.
+
+## `conda.nu`
+
+Limitations:
+
+- The "root_prefix" might not actually correspond to the correct path to the Conda envs. You can fix
```

Comment:
What exactly is meant by this? I cannot figure out what I should change to get a correct `$env.PATH` on a conda enviroment. I keep getting a `/usr/envs/cadquery/bin` when I have my enviroment inside `~/.conda/envs/cadquery/bin`.